jck/runtime.api/playlist.xml

```diff
@@ -41,8 +41,14 @@
 		<groups>
 			<group>jck</group>
 		</groups>
+		<disables>
+		    <disable>
+		      <comment>Disabled due to TKG/issues/123. Awaiting support in TKG to specify a path for parallelList.mk</comment>
+		    </disable>
```

Comment:
Please use the live issue URL as this is an open issue in TKG. No description is needed. We will need to check the URL for auto-reinclude. Also, it looks like space is used instead of tab. The indentation looks a little off.

Reply:
Updated.